Ilhasoft/chatguy-nlp
# Dev

- update tests docstring
- refactor ensemble_tests
- remove comments from app and try_except
- check if tests are working
